roles/kubernetes/master/templates/manifests/kube-apiserver.manifest.j2

```diff
@@ -42,6 +42,7 @@ spec:
     - --insecure-bind-address={{ kube_apiserver_insecure_bind_address }}
     - --bind-address={{ kube_apiserver_bind_address }}
     - --apiserver-count={{ kube_apiserver_count }}
+    - --endpoint-reconciler-type=lease
```

Comment:
This is a alpha feature in 1.9. You should add a check for 1.9 then:

`{% if kube_version | version_compare('v1.9', '>=') %}`